js/constants.js

```diff
@@ -4,6 +4,6 @@
 'use strict';
 
 module.exports = {
-  EXAMPLE_CONTEXT_URL: 'https://w3id.org/example/v1',
-  EXAMPLE_CBORLD_VALUE: 0x0 // value needs to be > 0x8000
+  CONTEXT_URL: 'https://w3id.org/security/ed25519-signature-2020/v1',
+  CBORLD_VALUE: 0x14
```

Comment:
Is there a more descriptive name that could be used for this before it spreads too widely?

Reply:
@davidlehn maybe `CODEC_VALUE`?

Reply:
@davidlehn Renamed to `CBORLD_CODEC_VALUE`